doc/dev/mds_internals/quiesce.rst

```diff
@@ -0,0 +1,90 @@
+MDS Quiesce Protocol
+====================
+
+The MDS quiesce protocol is a mechanism for "quiescing" (quieting) a subvolume
+in a file system, stopping all write (and most read) I/O.  A subvolume is a
+subtree in the file system marked with the ``ceph.dir.subvolume`` flag (like
+the ``ceph-mgr`` volumes plugin subvolumes).
```

Comment:
I'd like to decouple the generic quiesce mechanism that is technically applicable to any file system path, and the current limitation that requires the path to point to a snap realm without child realms, which is the case for subvolumes.

```suggestion
The MDS quiesce protocol is a mechanism for "quiescing" (quieting) a path
in a file system, stopping all write (and most read) I/O within the subtree.  
The current implementation limits the supported paths to the roots of subvolumes
created with the ``ceph-mgr`` volumes plugin.
```

Reply:
I added a note clarifying this.

Reply:
yes, done.